scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -312,16 +309,13 @@ def get_threshold_metrics(y_true, y_pred):
 ])
 
 
-# In[22]:
-
-def zero(w): return '{:.1%}'.format(np.mean(w == 0))
-def negative(w): return '{:,}'.format(np.sum(w < 0))
-def positive(w): return '{:,}'.format(np.sum(w > 0))
+# In[21]:
 
-coef_df.groupby('feature_set')['weight'].agg([zero, negative, positive])
+print('Signs of the coefficients')
+pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight))
 
 
-# In[23]:
+# In[22]:
 
 model = 'full'
```

Comment:
I don't see any benefit to defining this model variable. Just use `'full'` inline?